src/utils/dispatcher.js

```diff
@@ -2,16 +2,16 @@ import client from '../client';
 import { prefix, validCmds } from '../constants';
 
 export const parseMessage = (msg) => {
-  msg = msg.toString();
-  if (msg.substring(0, 2) === prefix) {
-    const args = msg.substring(2).split(' ');
+  const msgText = msg.toString();
```

Comment:
There has got to be a better way of doing this instead of just parsing and splitting.  I don't know if js has it but java has something called gson.  Allows a client to parse from json or string to an object.  I might be worth just deserializing into a class instead of manually parsing.

Reply:
There is not really a better way for Discord bots right now and your suggestion doesn't make sense in this context. We're literally just taking the message sent in Discord and making sure it has the prefix. There's a reason bots require prefixes as the whole system is based on the bot reading user messages.